src/styles/main.scss

```diff
@@ -1,3 +1,32 @@
-body {
+@import "variables";
+
+@import "normalize";
+
+.container {
   margin: 0;
+
+  height: 100vh;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($block-size, 1fr));
+}
+
+@media (min-width: 900px) {
+  .container {
+    grid-template-columns: repeat(3, minmax($block-size, 1fr));
+  }
+}
+
+.block {
+  min-height: $block-size;
+
+  display: grid;
+  justify-content: center;
+  align-items: center;
+
+  @each $num, $color in $colors {
+    &--#{$num} {
```

Comment:
@leny-jeanbaptiste please re-write using `:nth-child` - and rid of `block--` classes in html